CRITICAL - 2022-01-21 15:10:11 --> Undefined constant "CodeIgniter\Database\MySQLi\MYSQLI_STORE_RESULT"
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\Database.php(56): CodeIgniter\Database\Database->initDriver()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\Config.php(78): CodeIgniter\Database\Database->load()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(132): CodeIgniter\Database\Config::connect()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(825): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-01-21 15:11:54 --> Undefined constant "CodeIgniter\Database\MySQLi\MYSQLI_STORE_RESULT"
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\Database.php(56): CodeIgniter\Database\Database->initDriver()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\Config.php(78): CodeIgniter\Database\Database->load()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(132): CodeIgniter\Database\Config::connect()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(825): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-01-21 15:13:41 --> Undefined constant "CodeIgniter\Database\MySQLi\MYSQLI_STORE_RESULT"
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\Database.php(56): CodeIgniter\Database\Database->initDriver()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\Config.php(78): CodeIgniter\Database\Database->load()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(132): CodeIgniter\Database\Config::connect()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(825): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#12 {main}
CRITICAL - 2022-01-21 21:20:23 --> Call to undefined function CodeIgniter\locale_set_default()
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\spark(45): require('...')
#2 {main}
CRITICAL - 2022-01-21 21:20:49 --> Call to undefined function CodeIgniter\locale_set_default()
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\spark(45): require('...')
#2 {main}
CRITICAL - 2022-01-21 23:06:40 --> Call to undefined function CodeIgniter\locale_set_default()
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(28): require('...')
#2 {main}
CRITICAL - 2022-01-21 23:07:20 --> Call to undefined function CodeIgniter\locale_set_default()
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\spark(45): require('...')
#2 {main}
CRITICAL - 2022-01-21 23:09:06 --> Call to undefined function CodeIgniter\locale_set_default()
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\spark(45): require('...')
#2 {main}
ERROR - 2022-01-21 17:15:15 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-01-21 17:15:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-01-21 17:16:38 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-01-21 17:16:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-01-21 17:17:21 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-01-21 17:17:21 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-01-21 17:17:40 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-01-21 17:17:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-01-21 17:23:36 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-01-21 17:23:36 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-01-21 17:25:26 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-01-21 17:25:26 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-01-21 17:27:20 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-01-21 17:27:20 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\User->register()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#9 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#12 {main}
CRITICAL - 2022-01-21 18:36:51 --> {0} is not a valid rule.
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(300): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\User.php(36): CodeIgniter\Controller->validate()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\User->register()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-01-21 18:47:34 --> {0} is not a valid rule.
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(300): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\UserController.php(36): CodeIgniter\Controller->validate()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\UserController->register()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-01-21 18:57:57 --> {0} is not a valid rule.
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(300): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\UserController.php(36): CodeIgniter\Controller->validate()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\UserController->register()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-01-21 18:58:41 --> {0} is not a valid rule.
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(300): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\UserController.php(36): CodeIgniter\Controller->validate()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\UserController->register()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-01-21 19:00:42 --> CodeIgniter\Validation\Rules::is_unique(): Argument #2 ($field) must be of type string, null given, called in C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php on line 291
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(291): CodeIgniter\Validation\Rules->is_unique()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\app\Controllers\UserController.php(36): CodeIgniter\Controller->validate()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\UserController->register()
#5 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#6 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-01-21 19:07:21 --> Call to a member function getValue() on null
#0 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(828): App\Controllers\UserController->details()
#1 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(415): CodeIgniter\CodeIgniter->runController()
#2 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\CodeIgniter.php(323): CodeIgniter\CodeIgniter->handleRequest()
#3 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\Users\MouadCherkaoui\source\repos\codeigniter-4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('...')
#5 {main}
